Notebooks/FoFCatalogMatching Histogram.ipynb

```diff
@@ -1,296 +1,283 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Match truth and coadd catalogs for DC2 Run 1.1p\n",
+    "Owner: Yao-Yuan Mao, Anže Slosar, Bhairav Valera;  \n",
+    "Last Verified to Run: 2018-07-17  \n",
+    "*Note: as of 7/17, this notebook requires the master branch of gcr-catalogs! *\n",
+    "\n",
+    "## Learning objectives\n",
+    "After completing and studying this Notebook, you should be able to:\n",
+    "  1. Use GCR to load coadd catalog and truth catalog\n",
+    "  2. Use `filters` and `native_filters` appropriately\n",
+    "  3. Use `add_quantity_modifier` and `add_quantity_modifier`\n",
+    "  4. Use `FoFCatalogMatching` to do Friends-of-friends catalog matching\n",
+    "  5. Learn some cool Numpy tricks for binning, masking, and reshaping [Advanced]"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "#import sys\n",
-    "#sys.path.insert(0, '/global/common/software/lsst/common/miniconda/current/lib/python3.6/site-packages')\n",
-    "#sys.path.insert(0, '/path/to/your/cloned/gcr-catalogs')"
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from collections import defaultdict\n",
-    "import numpy as np\n",
-    "from astropy.coordinates import SkyCoord\n",
-    "import FoFCatalogMatching\n",
-    "from GCR import GCRQuery\n",
-    "import GCRCatalogs"
+    "import sys\n",
+    "sys.path.insert(0, '../../gcr-catalogs/')"
```

Comment:
What's the timescale for updating the default gcr-catalogs on NERSC to fix the need for this?
This specifically only works if someone has cloned the `gcr-catalogs` repo next to the `DC2_Repo`.